docker/config/local_dev.env

```diff
@@ -92,3 +92,11 @@ SECRET_KEY=secretkey
 STATSD_HOST=statsd
 SYMBOLS_BUCKET_DEFAULT_NAME=symbols
 SYMBOLS_BUCKET_EXCEPTIONS=
+
+# crontabber
+# ----------
+# In local development, it's useful to run the staging job set so we can run security package
+# checks.
+crontabber.jobs=socorro.cron.crontabber_app.STAGE_JOBS
```

Comment:
As of right now this is a superset of the non-stage jobs and should be fine, but maybe we don't intend that going forward?

Reply:
We need all the jobs in the jobs list available to run/test/develop-on in the local dev environment. When I do the `raw_adi` mover for -stage-new, we might want to change that, but it'll be tricky because I think we'll have two jobs with the same name.

I think what you did is fine. I'm not sure how to do this better in the confines of the current system.